cognite/client/_api/simulators/runs.py

```diff
@@ -175,6 +190,13 @@ def list(
                 ...     simulator_external_ids=["PROSPER", "DWSIM"],
                 ...     status="success"
                 ... )
+
+            Filter runs by time ranges:
+                >>> from cognite.client.data_classes.shared import TimestampRange
+                >>> res = client.simulators.runs.list(
+                ...     created_time=TimestampRange(min=0, max=1000000),
+                ...     simulation_time=TimestampRange(min=0, max=1000000),
```

Comment:
super nit, I think it would be slightly better if the timestamp values were realistic, to make it clear that it is millis and not seconds.
```suggestion
                ...     created_time=TimestampRange(min=0, max=1_700_000_000_000),
                ...     simulation_time=TimestampRange(min=0, max=1_700_000_000_000),
```